liby2util-r/src/y2log.cc

```diff
@@ -123,7 +123,12 @@ static int dup_stderr()
     }
     return 1;
 }
+/**
+ * silence warning:
+ *  'variable_not_used' defined but not used [-Wunused-variable]
+ */
 static int variable_not_used = dup_stderr();
+void y2log_unused_func() {if(variable_not_used){}}
```

Comment:
This is interesting, I am using `-Wall` which implies `-Wunused-variable` but I can only see the warning if there is an *error* elsewhere in the file that fails the compilation. If it compiles fine there is no such warning.

So I think this change is not necessary after all. If you insist, I'd prefer `__attribute__ ((unused))` for consistency with line 460.